src/gui/CMakeLists.txt

```diff
@@ -157,6 +157,13 @@ set(3rdparty_SRC
     ../3rdparty/qtsingleapplication/qtlocalpeer.cpp
     ../3rdparty/qtsingleapplication/qtsingleapplication.cpp
     ../3rdparty/qtsingleapplication/qtsinglecoreapplication.cpp
+    ../3rdparty/qmsgpack/src/msgpack.cpp
```

Comment:
I'm sure there is a nicer way ...

Reply:
I guess it's fine.

The ideal way would be to have that in a separate .cmake file included in the qmsgpack repository, but since it's not the case, then so be it.